src/app.types.tsx

```diff
@@ -272,3 +272,42 @@ export interface SessionListItem {
   timestamp: string;
   _id: string;
 }
+
+export interface FunctionTag {
+  type: 'channel' | 'functionToken' | 'function' | 'number' | 'string';
```

Comment:
Is `string` a valid function tag? I thought plain strings weren't allowed in functions